adapters/repos/db/bm25f_test.go

```diff
@@ -204,7 +205,17 @@ func TestBM25FJourney(t *testing.T) {
 		require.Equal(t, uint64(4), res[1].DocID())
 		require.Equal(t, uint64(5), res[2].DocID())
 		require.Equal(t, uint64(6), res[3].DocID())
-		require.Equal(t, uint64(2), res[4].DocID())
+	})
+
+	fmt.Println("Search with non alphanums")
```

Comment:
do we need this `fmt.Println` statement?

Reply:
This one, yes, it helps understand the output when a test fails.  It doesn't get printed unless the test fails